lnwallet/btcwallet/psbt_test.go

```diff
@@ -318,9 +318,13 @@ func TestSignPsbt(t *testing.T) {
 
 		// Let the wallet do its job. We expect to be the only signer
 		// for this PSBT, so we'll be able to finalize it later.
-		err = w.SignPsbt(packet)
+		signedInputs, err := w.SignPsbt(packet)
 		require.NoError(t, err)
 
+		// We expect one signed input at index 0
+		require.Equal(t, len(signedInputs), 1)
+		require.Equal(t, signedInputs[0], int32(0))
```

Comment:
nit: the expected value is the first argument and the actual value is the second (counting after the always present `t`). Otherwise the error message will be confusing. Also, could be using `EqualValues` to avoid needing to cast `0`.